front/src/applications/stdcm/components/StdcmForm/StdcmOrigin.tsx

```diff
@@ -28,7 +28,7 @@ const StdcmOrigin = ({ disabled = false, onItineraryChange }: StdcmItineraryProp
         disabled={disabled}
         onItineraryChange={onItineraryChange}
       />
-      <StdcmOpSchedule pathStep={origin} disabled={disabled} opId="origin-arrival" isOrigin />
+      <StdcmOpSchedule pathStep={origin} disabled={disabled} opId="origin-arrival" />
```

Comment:
We need to keep `isOrigin` otherwise we won't have the proper option in the select : 

![Capture d’écran 2025-03-06 à 12 26 13](https://github.com/user-attachments/assets/b08bc79e-003b-43e8-b3f8-e7db921edadc)

```suggestion
      <StdcmOpSchedule pathStep={origin} disabled={disabled} opId="origin-arrival" isOrigin/>
```

Reply:
I put it back, it was a mistake